exporter/streams_test.go

```diff
@@ -88,11 +88,11 @@ func TestStreamsGetStreamInfo(t *testing.T) {
 			if isNotTestTimestamp(info.LastGeneratedId) {
 				t.Errorf("Stream LastGeneratedId mismatch.\nActual: %#v;\nExpected any of: %#v", info.LastGeneratedId, TestStreamTimestamps)
 			}
-			if info.FirstEntryId != "" {
-				t.Errorf("Stream FirstEntryId mismatch.\nActual: %#v; - Expected empty", info.FirstEntryId)
+			if isNotTestTimestamp(info.FirstEntryId) {
```

Comment:
Why does this need to change?

Reply:
`exporter/exporter_test.go +110` Add `TestStreamTimestamps` to the stream
Since I have changed the method of getting the first and last entry id from a slice, when redis > v5.0, `getStreamInfo` should be able to get non-empty entry ids. 

https://redis.io/docs/latest/commands/xinfo-stream/